src/main/java/com/virtuslab/using_directives/custom/utils/TokenUtils.java

```diff
@@ -32,9 +32,7 @@ public static boolean isIdentifier(Tokens token) {
   }
 
   public static boolean isValidUsingDirectiveStart(Tokens token, Settings settings) {
-    return (settings.isAllowStartWithoutAt()
-            && (token == Tokens.USING || (settings.isAllowRequire() && token == Tokens.REQUIRE)))
-        || (settings.isAllowRequire() && token == Tokens.ATREQUIRE);
+    return (settings.isAllowStartWithoutAt() && token == Tokens.USING);
```

Comment:
It seems that `isAllowStartWithoutAt` is redundant now as well. It's the only allowed way if I understand correctly.

Reply:
Removed